src/components/shared/VirtualList.js

```diff
@@ -70,12 +98,27 @@ VirtualListInnerChunk.propTypes = {
   columnIndex: PropTypes.number.isRequired,
 };
 
-class VirtualListInner extends PureComponent {
+type VirtualListInnerProps = {
+  itemHeight: CssPixels,
+  className: string,
+  renderItem: RenderItem,
+  items: *[],
+  specialItems: *[],
```

Comment:
I don't mine which one of `Array<*>` or `*[]` you use but let's try to stay consistent at least in the same file :)